src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/blocklist.block.component.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    "use strict";
+
+    /**
+     * @ngdoc directive
+     * @name umbraco.directives.directive:umbBlockListBlock
+     * @description
+     * The component for a style-inheriting block of the block list property editor.
+     */
+    
+    angular
+        .module("umbraco")
+        .component("umbBlockListBlock", {
+            template: '<div ng-include="model.view"></div>',
+            controller: BlockListBlockController,
+            controllerAs: "model",
+            bindings: {
+                view: "@",
+                block: "=",
+                api: "=",
```

Comment:
Shouldn't this be a one way bind?

Reply:
I guess its the `api` you are referring too. Not so clear in this view. I have changed it to a one-way binding.
The block should be two way, to make it easy to make edits in custom views.